.github/workflows/regression-agility.yml

```diff
@@ -42,4 +42,4 @@ jobs:
           --env-var TOKEN_AUDIENCE=$token_audience \
           --env-var TOKEN_ENDPOINT=$token_endpoint \
           --env-var API_BASE_URL=$api_base_url \
-          --reporters cli,jsonname: Agility Tests
+          --reporters cli,json
```

Comment:
Shouldn't these names be preserved and passed in as an argument to the reporter as a title?

Reply:
IMO, no... we should just output JSON, and then sanitize and merge it, then build a report from that... we should not be doing what we are now.

Reply:
I believe the reporters also include the collection name as part of the output as well, so this would be redundant.

Reply:
If the reporter adds the collection name, then feel free to remove it.